sherpa/models/model.py

```diff
@@ -1376,25 +1371,31 @@ def rfunc(self, lhs):
 class ArithmeticModel(Model):
     """Support combining model expressions and caching results."""
 
-    cache = 5
-    """The maximum size of the cache."""
+    @property
+    def cache(self) -> int:
+        """The number of entires in the cache.
+
+        Changing this value will clear the cache.
```

Comment:
We *could* try to be clever and leave the existing cache alone of we change `.cache` to a larger value, but I do not think it worth the extra complexity (in code, but more importantly  documenting it / making it harder to understand what the rules are)/